### PR TITLE
Add more text on how the plugin automatically configures the CF depending on the version

### LINKED ISSUE #60: Updated instructions for JDK9+
Hi,
The current example to specify the CF version has the following:

    checkerFrameworkAnnotatedJDK 'org.checkerframework:jdk8:2.8.0'

I don't know how to adapt this to JDK 11 with CF 3.0.0, or if it's even needed.

### PR DESCRIPTION
**Pull request by kelloggm**
Fixes #60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,11 +104,22 @@ if (project.hasProperty("cfLocal")) {
     compileOnly files(cfHome + "/checker/dist/checker-qual.jar")
     testCompileOnly files(cfHome + "/checker/dist/checker-qual.jar")
     checkerFramework files(cfHome + "/checker/dist/checker.jar")
+    // only needed for JDK 8
     checkerFrameworkAnnotatedJDK files(cfHome + "/checker/dist/jdk8.jar")
   }
 }
 ```
 
+### JDK 8 vs JDK 9+
```
Comment (mernst) [inline README.md]: These are implementation details.  Therefore, I would omit them or state them at the very end of the README, after information that users are more likely to want.

File: README.md [hunk 2]
```diff
@@ -103,11 +104,22 @@ if (project.hasProperty("cfLocal")) {
...
+
+The plugin attempts to automatically configure the Checker Framework on both Java 8 and Java 9+ JVMs,
+following the [best practices in the Checker Framework manual](https://checkerframework.org/manual/#javac).
+In particular:
+* If both the JVM and target versions are 8, it applies the Java 8 annotated JDK.
+* If the JVM version is 9+ and the target version is 8 (and the Checker Framework
+version is >= 2.11.0), use the Error Prone javac compiler.
```
Comment (mernst) [inline README.md]: Minor:  Note that I changed "ErrorProne" to "Error Prone".

File: README.md [hunk 3]
```diff
@@ -103,11 +104,22 @@ if (project.hasProperty("cfLocal")) {
...
+* If the JVM version is 9+, use the `--add-opens` option to `javac`.
```
Comment (mernst) [inline README.md]: This is done even if the CF version is < 2.11?

Comment (kelloggm) [reply README.md]: Yes, but in that case it doesn't really matter, because the CF won't run.

File: README.md (hunk 1 above)
Comment (kelloggm) [reply README.md]: I'll move them to the end of the README.

**Pull request merged**